src/meta/src/stream/stream_manager.rs

```diff
@@ -458,16 +473,49 @@ impl GlobalStreamManager {
             table_fragments.internal_table_ids().len() + mv_table_id.map_or(0, |_| 1)
         );
         revert_funcs.push(Box::pin(async move {
-            if create_type == CreateType::Foreground {
-                if let Err(e) = hummock_manager_ref.unregister_table_ids(&registered_table_ids).await {
-                    tracing::warn!("Failed to unregister compaction group for {:#?}. They will be cleaned up on node restart. {:#?}", registered_table_ids, e);
-                }
+            if let Err(e) = hummock_manager_ref.unregister_table_ids(&registered_table_ids).await {
```

Comment:
Still need to check create type for it. FYI, we don't support create sink in background for now, but we still need take this feature into consideration after it get supported.